prqlc/prqlc/src/codegen/ast.rs

```diff
@@ -378,11 +379,11 @@ fn valid_prql_ident() -> &'static Regex {
     })
 }
 
-pub fn maybe_escape_ident_part(s: &str) -> String {
+pub fn write_ident_part(s: &str) -> Cow<str> {
```

Comment:
(I'm surprised rust doesn't use `Cow` more for strings, seems like the best of both worlds...)